prime-router/settings/organizations.yml

```diff
@@ -1087,16 +1087,16 @@
       qualityFilter:
         - "true"
       processingModeFilter: [ "true" ]
-      mappedConditionFilter: # this filter will only pass observations stamped with condition codes AOE or 840539006
-        - type: codeString
-          value: 840539006,AOE
+      mappedConditionFilter: []
```

Comment:
Should this be commented out?

Reply:
make it an empty list since the mapopedConditionFilter work is not merged yet?
the current:
```
mappedConditionFilter: # this filter will only pass observations stamped with condition codes AOE or 840539006
        - type: codeString
          value: 840539006,AOE
```

e.g. this is what I got from Gabriel:

```
mappedConditionFilter:
        - type: conditionCode
          value: 840539006,1234
        - type: conditionName
          value: flu,polio
        - type: conditionFhirExpression
          value: "Bundle.identifier.value.empty().not()"
          defaultResponse: true # not required
          reverseFilter: false # not required
```

We can come back to this after the mappedConditionFilter is in master

Reply:
I think it is in `master`?

Reply:
just sync with master and will make some adjustment

Reply:
now mappedConditionFilter in ./prime-router/settings/organizations.yml has been recovered, and the settings has been serialized/de-serialized successsfully